docs/pages/cheatsheet.adoc

```diff
@@ -596,12 +661,6 @@ Removed from service object.
 |resource       | access name           | The name of an access where the returned transfer object is available.
 |====================
 
-[options="header", cols="1,2"]
-|====================
-|Annotation | Meaning
-|@Save      | Generates the default save behavior.
-|====================
-
 == Row
 
 *Syntax:*
```

Comment:
Row means record? Row is directly mapped to access, so it can be used in non human access to descibe list type accesses? In the case I think record is better name, because my first association is table row which is UI element is JSL.

Reply:
Your first guess is correct. It is a row of a table.

Reply:
So the second guess is also true?

Reply:
My original proposal was "record". Norbert convinced me that "row" is more "natural" from ui point of view.

Reply:
But UI is just one layer. And it is related to Data, it is used in Transfer layer and is addressable by System actors too.

Reply:
This is supposed to be the "View" section, not the Transfer section. Transfer is for systems, and View is for humans. I didn't know that we would use the same terminology for both layers. Having Record as a super type of Row sounds good to me.

Reply:
No. Row is after the view section. Transfer objects cannot refer to rows. Only views can refer to rows.